src/components/number_input.tsx

```diff
@@ -41,6 +42,16 @@ const NumberInput: FC<NumberInputProps> = ({
 }): any => {
     const [error, setError] = useState<string>('')
 
+    // use the cursor postion when user edits the data in the component
+    const [cursor, setCursor] = useState<number | null>(null)
+    const ref = useRef<HTMLInputElement>(null)
+
+    // Refresh after the first render and every time the component updates
+    useEffect(() => {
+        const input = ref.current
+        if (!!input) input.setSelectionRange(cursor, cursor)
```

Comment:
My bad. Can you change `!!input` to `input`?  I put the !! in when I was trying to fix the Typescript error - it didn't help.

Reply:
Done